src/pybind/mgr/prometheus/module.py

```diff
@@ -3,20 +3,19 @@
 from collections import defaultdict
 import json
 import math
-import os
 import re
 import threading
 import time
 import enum
 from collections import namedtuple
+from collections import OrderedDict
```

Comment:
Oh, so the discrepancy between `main` and the backport has been tackled not by adding the `Conflict` section but rather by squeezing the diff. Fine!